### PR TITLE
Add update time and cur pkt versions to base json info

### PR DESCRIPTION
**Pull request by dsurnin**
Signed-off-by: dsurnin <dmitry.surnin@dsr-corporation.com>

### CODE REVIEW

File: indy_node/test/validator_info/test_validator_info.py [hunk 1]
```diff
@@ -187,3 +186,12 @@ def test_validator_info_file_metrics_count_all_ledgers_field_valid(node):
...
     assert has_cnt == len(new_ids) + 3
+
+
+def test_validator_info_update_date_field_valid(info):
+    assert "Update time" in info
+    import time
+    import datetime
+    from_str = time.mktime(datetime.datetime.strptime(info["Update time"],
+                                                      "%A, %B %d, %Y %I:%M:%S %p %z").timetuple())
+    assert int(from_str) == info["timestamp"]
```
Comment (ashcherbakov) [inline indy_node/test/validator_info/test_validator_info.py]: What is the difference between "Update Time" and "timestamp"?

File: indy_node/server/validator_info_tool.py [hunk 2]
```diff
@@ -11,9 +13,12 @@ class ValidatorNodeInfoTool(PlenumValidatorNodeInfoTool):
     @property
     def info(self):
         info = super().info
+        ts_str = "{}".format(time.strftime(
+            "%A, %B %{0}d, %Y %{0}I:%M:%S %p %z".format('#' if os.name == 'nt' else '-'),
+            time.localtime(info["timestamp"])))
+        info.update({"Update time": ts_str})
         if 'Node_info' in info:
             if 'Metrics' in info['Node_info']:
-                info['Node_info']['Metrics']['transaction-count'].update(config=self.__config_ledger_size)
```
Comment (ashcherbakov) [inline indy_node/server/validator_info_tool.py]: Why did we remove this?

File: indy_node/test/validator_info/test_validator_info.py (hunk 1 above)
Comment (dsurnin) [reply indy_node/test/validator_info/test_validator_info.py]: timestamp is an int value
Update Time is human readable string with tz offset

File: indy_node/server/validator_info_tool.py (hunk 2 above)
Comment (dsurnin) [reply indy_node/server/validator_info_tool.py]: it was moved to plenum

**Pull request merged**